conbench/api/index.py

```diff
@@ -9,11 +10,21 @@
 from ..api import api, rule
 from ..api._docs import spec
 from ..api._endpoint import ApiEndpoint
-from ..db import Session
+from ..config import Config
+from ..db import Session, empty_db_tables
+
+log = logging.getLogger(__name__)
 
 
 @api.route("/docs.json")
 def docs():
+    d = spec.to_dict()
+
+    # In TESTING mode there is a special endpoint that we do not want to mess
+    # with test_docs.
+    if "/api/wipe-db" in d["paths"]:
+        del d["paths"]["/api/wipe-db"]
+
     return f.jsonify(spec.to_dict())
```

Comment:
You wanted to `return f.jsonify(d)`, no?

Reply:
Yes. Thank you for your pair of eyes!!

Question is: why did the tests pass nevertheless? :D Looking now.